README.md

```diff
@@ -9,4 +9,7 @@ Check out the the subfolders for more information on each brick.
 ## Bricks
 
 ### [wnma_dart_package](wnma_dart_package/README.md)
-A template for a new Dart package including the option to have subpackages.
\ No newline at end of file
+A template for a new Dart package including the option to have subpackages.
```

Comment:
why double headline?

Reply:
One of them is for the Dart package, the other for the flutter package. This README could be made a little nicer, I agree 😅